lib/ex_uid2/dsp.ex

```diff
@@ -8,10 +8,13 @@ defmodule ExUid2.Dsp do
   @keyring :uid2_decryption_keyring
   @refresh_rate_ms 5000
 
+  @default_opts [mode: :active]
+
   require Logger
 
   def start_link(opts \\ []) do
-    GenServer.start_link(__MODULE__, opts, name: __MODULE__)
+    genserver_opts = Keyword.merge(@default_opts, opts)
+    GenServer.start_link(__MODULE__, genserver_opts, name: __MODULE__)
   end
```

Comment:
I think an anti-pattern that I'm definitely guilty of is confusing 'arguments' to a new GenServer, and the options passed to `GenServer.start_link/3`.

Given our discussion yesterday, it's probably fine to hardcode the start_link/3 opts to `[name: __MODULE__]`, but if you revisit your decision to allow for multiple Dsp servers to exist at the same time, then you may want to add an argument to this function to allow for specifying different GenServer options, e.g. name registration, timeouts, etc.